seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -552,6 +598,118 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_s
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, '/login/google-oauth2?next=/api/create_project_from_workspace/my-seqr-billing/anvil-no-project-workspace2/submit')
 
+    @mock.patch('seqr.models.Project._compute_guid', lambda project: f'P_{project.name}')
```

Comment:
why would we need to mock project guid, you should never create a project in this tes?